### PR TITLE
MITRE: Changes to mapping from Command and Control

### PR DESCRIPTION
**Pull request by mikykeane**
This PR aims to tackle the issue from #803 

The following MITRE mappings have been changed:

- Rule ID: **5706**  `sshd: insecure connection attempt`  

> Removed
> **T1043** Deprecated. (**Commonly Used Port**). 
> 
> Added:
>  **T1021** **Remote Services**  -  **Lateral Movement**.
> 
> 

- Rule ID: **11209** `proftpd: Attempt to bypass firewall that can't adequately keep state of FTP traffic`

> Removed:  
> **T1043** Deprecated. (**Commonly Used Port**)  - **Command and Control**
> **T1065 Uncommonly Used Port**  - **Command and Control**

> Added: 
> **T1071**   **Application Layer Protocol**  - **Command and Control**
> 

- Rule ID: **11512** `Multiple FTP errors from the same source`

> Removed:
> **T1043** Deprecated. (**Commonly Used Port**)  - **Command and Control**
> 
> Added:
> **T1071**   **Application Layer Protocol**  - **Command and Control**
>  

- Rule ID: **82405** `Potential malware` 

> Removed:
> **T1043** Deprecated. (**Commonly Used Port**)  - **Command and Control**
> 
> Added:
> **T1071**  **Application Layer Protocol**  - **Command and Control**
> **T1557**  **Man-in-the-Middle** -  **Credential Access**
> 

- Rule ID: **4325** `PIX: ARP collision detected`

> Added:
> **T1557**  **Man-in-the-Middle** -  **Credential Access**
> 

- Rule ID: **7209** `Arpwatch: Possible arpspoofing attempt`

> Added:
> **T1557**  **Man-in-the-Middle** -  **Credential Access**
> 

- Rule ID: **24061** `ARP cache:`

> Added:
> **T1557**  **Man-in-the-Middle** -  **Credential Access**
> 
> 

- Rule ID: **64011** `ASA: ARP collision detected`

> Added:
> **T1557**  **Man-in-the-Middle** -  **Credential Access**
> 
> 

The following are related to mapping of **T1102** **Web Service** - The concept is misunderstood. The technique refers to attacks done via Web services, NOT aimed at Web Services.

- Rule ID: **9510**  `Attack against Wordpress detected.`
- Rule ID: ** 30107** `Apache: Code Red attack.`
- Rule ID: ** 30307** `Apache: Client sent malformed Host header. Possible Code Red attack.`
- Rule ID: ** 31411** `PHP web attack.`

> Removed:
> **T1102** **Web Service**
> 

- Rule ID: **12101** `Invalid DNS packet. Possibility of attack.`

> Added:
> **T1557**  **Man-in-the-Middle** -  **Credential Access**
> 
> 

- Rule ID: ** 81610**  `Fortigate: Default tunneling setting. Could be IPS.`

> Added:
> **T1572**  **Protocol Tunneling** - **Command and Control**
> 

- Rule ID: ** 81611** `Fortigate: Multiple default tunneling setting events from same source.`

> Removed:
> **T1071**   **Application Layer Protocol**  - **Command and Control**
> 
> Added:
> **T1572**  **Protocol Tunneling** - **Command and Control**
> 

- Rule ID: ** 82405** `Cisco eStreamer: APP-DETECT DNS request for potential malware`

> Removed:
> **T1043** Deprecated. (**Commonly Used Port**)  - **Command and Control**
> 
> Added:
> **T1557**  **Man-in-the-Middle** -  **Credential Access**
> 

- Rule ID: ** 24215** `osquery: $(osquery.pack) $(osquery.subquery): MaMi malware detected, infected DNS address $(osquery.columns.address)`

> Added:
> **T1557**  **Man-in-the-Middle** -  **Credential Access**
> 

**NOTE**

The following mappings, have not yet been added to the framework: **T1557  Man-in-the-Middle** and   **T1572**  **Protocol Tunneling**

Therefore, they will not yet show in the dashboards until they are correctly added.

### CODE REVIEW

File: rules/0190-ms_ftpd_rules.xml [hunk 1]
```diff
@@ -71,7 +71,7 @@
     <description>MS-FTP: Multiple FTP errors from same source.</description>
     <mitre>
       <id>T1021</id>
-      <id>T1043</id>
+      <id>T1071</id>
```
Comment (fabamatic) [inline rules/0190-ms_ftpd_rules.xml:74]: T1071.002 "Application Layer Protocol: File Transfer Protocols" looks more accurate

File: rules/0250-apache_rules.xml [hunk 2]
```diff
@@ -61,9 +61,6 @@
     <description>Apache: Code Red attack.</description>
     <info type="link">http://www.cert.org/advisories/CA-2001-19.html</info>
     <info type="text">CERT: Advisory CA-2001-19 "Code Red" Worm Exploiting Buffer Overflow In IIS Indexing Service DLL</info>
-    <mitre>
-      <id>T1102</id>
-    </mitre>
     <group>automatic_attack,pci_dss_6.2,pci_dss_6.5.2,pci_dss_11.4,gdpr_IV_35.7.d,nist_800_53_SI.2,nist_800_53_SA.11,nist_800_53_SI.4,tsc_CC6.8,tsc_CC6.6,tsc_CC7.1,tsc_CC6.1,tsc_CC7.2,tsc_CC7.3,</group>
```
Comment (fabamatic) [inline rules/0250-apache_rules.xml:64]: For this one the tags T1190 "Exploit Public-Facing Application" and T107.001 "Application Layer Protocol: Web Protocols" may be correct

File: rules/0250-apache_rules.xml [hunk 3]
```diff
@@ -221,9 +218,6 @@
     <description>Apache: Client sent malformed Host header. Possible Code Red attack.</description>
     <info type="link">http://www.cert.org/advisories/CA-2001-19.html</info>
     <info type="text">CERT: Advisory CA-2001-19 "Code Red" Worm Exploiting Buffer Overflow In IIS Indexing Service DLL</info>
```
Comment (fabamatic) [inline rules/0250-apache_rules.xml:220]: Same as above: For this one the tags T1190 "Exploit Public-Facing Application" and T107.001 "Application Layer Protocol: Web Protocols" may be correct

File: rules/0265-php_rules.xml [hunk 4]
```diff
@@ -59,9 +59,6 @@
     <match> expects parameter 1 to be string, array given in</match>
     <group>attack,pci_dss_6.5,pci_dss_11.4,gdpr_IV_35.7.d,nist_800_53_SA.11,nist_800_53_SI.4,tsc_CC6.6,tsc_CC7.1,tsc_CC8.1,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
     <description>PHP web attack.</description>
```
Comment (fabamatic) [inline rules/0265-php_rules.xml:61]: Probably this one also pairs with  tags T1190 "Exploit Public-Facing Application" and T107.001 "Application Layer Protocol: Web Protocols"

File: rules/0390-fortigate_rules.xml [hunk 5]
```diff
@@ -105,9 +105,6 @@ ID: 81600 - 80799
         <if_matched_sid>81610</if_matched_sid>
         <same_source_ip />
         <description>Fortigate: Multiple default tunneling setting events from same source.</description>
```
Comment (fabamatic) [inline rules/0390-fortigate_rules.xml:107]: Not sure about this one: T1562.004 Impair Defenses: Disable or Modify System Firewall

File: rules/0425-cisco-estreamer_rules.xml [hunk 6]
```diff
@@ -51,8 +51,7 @@
         <id>APP-DETECT</id>
         <match>DNS request for potential malware</match>
         <mitre>
-          <id>T1071</id>
-          <id>T1043</id>
+            <id>T1071</id>
```
Comment (fabamatic) [inline rules/0425-cisco-estreamer_rules.xml:54]: T1071.004 "Application Layer Protocol: DNS" is more accurate

**Review by fabamatic — CHANGES_REQUESTED** [review]

**Comment by jcruzlp** [comment]
These changes were added in this PR https://github.com/wazuh/wazuh/pull/9194

**Pull request closed without merging**